packages/beacon-node/src/api/rest/base.ts

```diff
@@ -43,7 +43,15 @@ export class RestApiServer {
     const server = fastify({
       logger: false,
       ajv: {customOptions: {coerceTypes: "array"}},
-      querystringParser: querystring.parse,
+      querystringParser: (str) =>
+        qs.parse(str, {
+          // defaults to 20 but Beacon API spec allows max items of 30
+          arrayLimit: 30,
```

Comment:
Beacon API spec defines `30` as `maxItems` for some values, e.g. `getStateValidators` - `id` query, see [spec value](https://github.com/ethereum/beacon-APIs/blob/f087fbf2764e657578a6c29bdf0261b36ee8db1e/apis/beacon/states/validators.yaml#L23)